_posts/newsletters/2019-10-24-newsletter.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: October 2019 US-RSE Newsletter
+subtitle: US-RSE Newsletter
+category: newsletter
+tags: [newsletter, PEARC19, Learning Lab, Code of Conduct, SC19, stories]
+---
+
+
+In this bi-monthly newsletter, we will share recent, current and
+planned activities of the US-RSE Association. Newsletters will be
+available on our [website](https://us-rse.org/newsletters/) beside the
+growing resources and information on the US-RSE Association. A sign-up
+option for our newsletter is available
+[here](https://us-rse.org/join/).
+
+
+## Report from PEARC19 
+
+The BoF at PEARC19 was well attended with about 30
+people. A main part of the discussion was about the definition of
+RSEs. We would like to stay very inclusive, with members who see
+themselves as RSEs and also people who are interested in supporting
+the goals of US-RSE and RSE associations in general or are just
+interested in knowing about efforts around the sustainability of
+research software. The discussion revolved around beneficial
+activities for the community and what US-RSE could provide. Feedback
+included the need for sharing best and bad practices, lobbying RSE
+positions to executives, networking, advertising of open jobs,
+mentorship programs, platform for sharing expertise, interactions
+between academia, industry and national labs and support organizations
+in incentivizing good software engineering practices.
+
+
+![alt text]({{ site.baseurl }}/assets/posts/2019-10-24-PEARC19-BOF.png "US-RSE BOF at PEARC19")
+
+![alt text]({{ site.baseurl
+ }}/assets/posts/2019-10-24-RSE-definition-missing.png "Responses to
+ the missing componenets of the US-RSE Definition during BOF at
+ PEARC19")
+Results from PEARC19 interactive audience question “what is missing in the US-RSE definition?”
+
+
+## Learning Lab at Gateways 2019
+
+[Gateways 2019](https://sciencegateways.org/web/gateways2019) took
+place in September in San Diego and the conference offered Learning
+Labs - conceptual a mixture of BoFs and break-out sessions - to spawn
+discussions about topics suggested by participants. Sandra Gesing
+suggested a learning lab about US-RSE and the lively discussion was
+mainly around how US-RSE can support career paths in the US and help
+with creating positions at institutions. One take-away is that
+templates for job descriptions would be helpful for managers offering
+RSE-like positions.
```

Comment:
I have the one hat Ruth and I wrote in 2016, I could see if I could share it. (Where?)

Reply:
I have a bunch too.  We'll need to figure it out (and any ramifications to sharing)